apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -310,7 +310,7 @@ public override Task<int> ExecuteNonQueryAsync(CancellationToken cancellationTok
                     Table = SpannerCommandTextBuilder.TargetTable,
                     Columns =
                     {
-                        Parameters.Cast<SpannerParameter>()
+                        Parameters
```

Comment:
Unwrap these changes to single lines, given that they're shorter now? (Not overly bothered, just worth considering.)